doc/contribute/doc_guidelines.rst

```diff
@@ -0,0 +1,378 @@
+.. _doc_guidelines:
+
+Documentation Guidelines
+########################
+
+Project ACRN content is written using the `reStructuredText`_ markup
+language (``.rst`` file extension) with Sphinx extensions, and processed
+using Sphinx to create a formatted standalone website.  Developers can
+view this content either in its raw form as .rst markup files, or (with
+Sphinx installed) they can build the documentation using the Makefile
+(on Linux systems) to
+generate the HTML content. The HTML content can then be viewed using a
+web browser. This same ``.rst`` content is also fed into the
+`Project ACRN documentation`_ website.
+
+You can read details about `reStructuredText`_
+and about `Sphinx extensions`_ from their respective websites.
+
+.. _Sphinx extensions: http://www.sphinx-doc.org/en/stable/contents.html
+.. _reStructuredText: http://docutils.sourceforge.net/docs/ref/rst/restructuredtext.html
+.. _Sphinx Inline Markup:  http://sphinx-doc.org/markup/inline.html#inline-markup
+.. _Project ACRN documentation:  http://projectacrn.github.io
+
+This document provides a quick reference for commonly used reST and
+Sphinx-defined directives and roles used to create the documentation
+you're reading.
+
+Headings
+********
+
+While reST allows use of both and overline and matching underline to
+indicate a heading, we only use an underline indicator for headings.
+
+* Document title (h1) use "#" for the underline character
+* First section heading level (h2) use "*"
```

Comment:
I think this is by "design" but I'll ask anyway because it confused me the first time I read it... the *first* section heading is referred to as `h2`... correct?

Reply:
I'm using the HTML tag references here, which likely is the cause for confusion.  I'll just remove the HTML references.